python/tvm/tir/sparse.py

```diff
@@ -28,7 +28,20 @@
 
 class Axis(Object):
     """Base class of all the sparse axes."""
+    @property
+    def name(self):
+        return _ffi_api.GetAxisName(self)
+    
+    @property
+    def length(self):
+        return _ffi_api.GetAxisLength(self)
+    
+    @property
+    def idtype(self):
+        return _ffi_api.GetAxisIndexType(self)
```

Comment:
To enable these properties, we should register the FFI functions first.

An example might be
https://github.com/yzh119/tvm/blob/d23688c3e5aa51e69d13fa469abeca3c9e054714/python/tvm/tir/schedule/schedule.py#L171-L184
https://github.com/yzh119/tvm/blob/6b3fe95f3c5ff75f65d15d6fc2797f584c456a5a/src/tir/schedule/schedule.cc#L47-L52